Formula/tezos-baker-008-PtEdo2Zk.rb

```diff
@@ -2,23 +2,124 @@
 #
 # SPDX-License-Identifier: LicenseRef-MIT-TQ
 
-require File.join(File.dirname(__FILE__), "..", "FormulaAbstract", "tezos")
+class TezosBaker008Ptedo2zk < Formula
+  @all_bins = []
 
-class TezosBaker008Ptedo2zk < Tezos
-  init
+  class << self
+    attr_accessor :all_bins
+  end
+  homepage "https://gitlab.com/tezos/tezos"
+
+  url "https://gitlab.com/tezos/tezos.git", :tag => "v8.2", :shallow => false
+
+  version "v8.2-3"
+
+  build_dependencies = %w[pkg-config autoconf rsync wget opam rust]
+  build_dependencies.each do |dependency|
+    depends_on dependency => :build
+  end
+
+  dependencies = %w[gmp hidapi libev libffi]
+  dependencies.each do |dependency|
+    depends_on dependency
+  end
+  depends_on "tezos-sapling-params"
```

Comment:
Why not add this to the `dependencies` list just above instead?

Same in `tezos-client` and `tezos-node`